modules/openapi-generator/src/main/resources/dart-dio/class.mustache

```diff
@@ -10,8 +10,6 @@ abstract class {{classname}} implements Built<{{classname}}, {{classname}}Builde
     {{#isNullable}}
         @nullable
     {{/isNullable}}
-
-    {{#description}}/* {{{description}}} */{{/description}}
```

Comment:
Why remove the class description?

Reply:
This one was a duplicate, it exists at the top of the file